backend/admin.py

```diff
@@ -0,0 +1,10 @@
+from django.contrib import admin
+from canvas-app-explorer.backend.models import LtiTool, CanvasPlacement

```

Comment:
This import should just be

```suggestion
from backend.models import LtiTool, CanvasPlacement

```